sdk/tests/deqp/modules/shared/glsBuiltinPrecisionTests.js

```diff
@@ -5069,20 +5092,8 @@ var setParentClass = function(child, parent) {
      * @constructor
      * @extends {glsBuiltinPrecisionTests.ArcTrigFunc}
      */
-    glsBuiltinPrecisionTests.ASin = function() {
-        glsBuiltinPrecisionTests.ArcTrigFunc.call(this, 'asin', Math.asin, 4,
-                                                tcuInterval.withNumbers(-1, 1),
-                                                tcuInterval.withNumbers(-Math.PI * 0.5, Math.PI * 0.5));
-    };
-
-    setParentClass(glsBuiltinPrecisionTests.ASin, glsBuiltinPrecisionTests.ArcTrigFunc);
-
-    /**
-     * @constructor
-     * @extends {glsBuiltinPrecisionTests.ArcTrigFunc}
-     */
     glsBuiltinPrecisionTests.ACos = function() {
-        glsBuiltinPrecisionTests.ArcTrigFunc.call(this, 'acos', Math.acos, 4,
+        glsBuiltinPrecisionTests.ArcTrigFunc.call(this, 'acos', Math.acos, 4096.0,
```

Comment:
The native code relaxed the restriction of the precision by this code snippet: https://android.googlesource.com/platform/external/deqp/+/deqp-dev/modules/glshared/glsBuiltinPrecisionTests.cpp#2425